web/app.fluidity.money/app/routes/$network/query/volumeStats.tsx

```diff
@@ -0,0 +1,52 @@
+import { json, LoaderFunction } from "@remix-run/node";
+import BN, { BN } from "bn.js";
+import { useVolumeCalculation, VolumeCalculationResponse } from "~/queries/useVolumeCalculation";
+import config from "~/webapp.config.server";
+
+export const loader: LoaderFunction = async ({ params }) => {
+    const { network } = params;
+
+    // Postprocess res
+    const fdaiPostprocess = (value: VolumeCalculationResponse) => {
```

Comment:
feels kind of bad having this hardcoded for fDAI

Reply:
:man_shrugging: